app/controllers/nodes_controller.rb

```diff
@@ -6,6 +6,16 @@ class NodesController < ApplicationController
   def show
   end
 
+  # POST /nodes/1/cordon
+  def cordon
+    should_cordon = !params[:unset]
```

Comment:
how to read this logic? I don't quite understand
is params[:unset] a shortcut in rails for something?

Reply:
I've improved the readability of this code. Please check.

Basically if `unset` parameter is being set, it uncordons the node instead of cordoning it.